packages/sdk-core/src/BacktraceCoreClient.ts

```diff
@@ -418,4 +420,51 @@ export abstract class BacktraceCoreClient<
     private static destroy() {
         this._instance = undefined;
     }
+
+    private validateAttributes() {
+        function isNotEmptyString(v: unknown) {
```

Comment:
we're introducing a nice way to validate attributes, but in our scenario, we don't need such complexity. What we can do is just simply test if those two attributes are defined. Based on what we want to verify and how easily we can achieve that, wouldn't it be better if we keep the smallest/simplest solution for now?

Reply:
how would you do this more "easily"?

Reply:
just simply 
```
const attributes = this.attributeManager.ger()
const validAttributes = !!attributes["application"] && !!attributes["application.version"]
```

Reply:
simplified a bit